RecoMuon/TrackerSeedGenerator/src/SeedMvaEstimator.cc

```diff
@@ -39,19 +42,18 @@ void SeedMvaEstimator::getL1MuonVariables(const GlobalVector& global_p,
                                           const l1t::MuonBxCollection& l1Muons,
                                           float& dR2dRL1SeedP,
                                           float& dPhidRL1SeedP) const {
-  for (int ibx = l1Muons.getFirstBX(); ibx <= l1Muons.getLastBX(); ++ibx) {
-    if (ibx != 0)
-      continue;  // -- only take when ibx == 0 -- //
-
-    for (auto it = l1Muons.begin(ibx); it != l1Muons.end(ibx); it++) {
-      if (it->hwQual() < minL1Qual_)
-        continue;
-
-      float dR2tmp = reco::deltaR2(it->etaAtVtx(), it->phiAtVtx(), global_p.eta(), global_p.phi());
-      if (dR2tmp < dR2dRL1SeedP) {
-        dR2dRL1SeedP = dR2tmp;
-        dPhidRL1SeedP = reco::deltaPhi(it->phiAtVtx(), global_p.phi());
-      }
+  int ibx = 0;  // -- only take when ibx == 0 -- //
+
+  auto eta = global_p.eta();
+  auto phi = global_p.barePhi();
```

Comment:
Is there any implication in moving from phi() to barePhi() ?

Reply:
absolutely equivalent in this contest

Reply:
see https://cmssdt.cern.ch/lxr/source/DataFormats/Math/interface/deltaR.h#0036